scripts/jenkins/qa_openstack.sh

```diff
@@ -19,8 +19,12 @@ ifconfig | grep inet
 
 # setup optional extra disk for cinder-volumes
 dev_cinder=/dev/vdb
-if ! test -e $dev_cinder && file -s /dev/sdb|grep -q "ext3 filesystem data" ; then
-    dev_cinder=/dev/sdb
+if ! test -e $dev_cinder ; then
+    # maybe we run under xen
+    dev_cinder=/dev/xvdb
+    if ! test -e $dev_cinder && file -s /dev/sdb|grep -q "ext3 filesystem data" ; then
+        dev_cinder=/dev/sdb
+    fi
```

Comment:
I am ok with this, but I wonder why not to use a more simple approach here, like

```bash
if test -e /dev/vdb; then
  dev_cinder=/dev/vdb
elif ...
  ...
else
  echo "Extra disk for cinder-volumes not detected"
  exit -1 # if this an error?
fi
```